.aspell.en.pws

```diff
@@ -1,96 +1,164 @@
-personal_ws-1.1 en 89 
+personal_ws-1.1 en 163 
 APIs
 ActiveMQ
 Ansible
+Bitbucket
 BlazeGraph
 Blazegraph
 CLA
+CLI
+CSS
+CentOS
 Chullo
 DNS
 Discoverygarden
+Djatoka
+Dockerfile
+Dockerfiles
+Dockerhub
 Drupal's
 Drush
 FOAF
 FOXML
+FQDN
 Flysystem
 GPLv
+GSearch
+Grinnell
+Holyoke
+ICG
 IIIF
+ISLE's
 ImageMagick
 Islandora's
+Islandoracon
 JPG
 JPGs
-JSON
```

Comment:
This is more of an aside then anything, as it shouldn't hold up this pr.  We must be running aspell in case insensitive mode?  Just wondering how we can have this file be used consistently by folks.